lambda/handler/uber_handler.py

```diff
@@ -50,3 +51,18 @@ def confirm_requests(event, context):
     body = {"requests": requests}
     response = {"statusCode": 200, "body": json.dumps(body)}
     return response
+
+
+def create_poll(event, context):
+    # ping auth0 API with token
+    logger.info(f"event: {event}")
+
```

Comment:
Please make it so only ubermentors can create polls. See the other handlers in uber_handler.py